code/index.html

```diff
@@ -16,25 +16,33 @@
 
       <img
         class="guess-who-icon"
-        src="images/guess-who-bubble.png"
+        src="./images/guess-who-bubble.png"
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1 class="find-out-title">Does the person have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
+          <option selected disabled default>---</option>
```

Comment:
This small deatail was nice! Well done

Reply:
Yes, but it should be before the hair group to make more sense. 😺